city_scrapers/spiders/chi_ssa_4.py

```diff
@@ -0,0 +1,172 @@
+import json
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa4Spider(CityScrapersSpider):
+    name = "chi_ssa_4"
+    agency = "Chicago Special Service Area #4 South Western Avenue"
+    timezone = "America/Chicago"
+    start_urls = [
```

Comment:
You'll want to dynamically generate this so that the time range will pull the few months around the time it's being run. Here's an example of something like that:

https://github.com/City-Bureau/city-scrapers/blob/9b90ad7ac653c608ccb84b0d0b7f5dc183405b35/city_scrapers/spiders/il_public_health.py#L17-L28

Reply:
I think the start_urls doesn't need to be dynamically generated actually. The start URL doesn't actually affect our parsing at all (we should only be parsing the HTML we get from POST responses).

What we can do instead is dynamically determine the start date for our POST request (it'll only respond with meetings after that date) and only parse X months into the future. I'll do that